foxglove/console/api.go

```diff
@@ -44,8 +46,8 @@ type StreamRequest struct {
 }
 
 func (req *StreamRequest) Validate() error {
-	if req.RecordingID == "" && req.ImportID == "" && req.DeviceID == "" && req.DeviceName == "" {
-		return fmt.Errorf("either recording-id, import-id, or all three of device-id/device-name, start, and end are required")
+	if req.RecordingID == "" && req.ImportID == "" && req.DeviceID == "" && req.DeviceName == "" && req.Key == "" {
+		return fmt.Errorf("either recording-id/recording-key, import-id, or all three of device-id/device-name, start, and end are required")
```

Comment:
Should "recording-key" be renamed "key" to match the arg?

Reply:
I see you dropped explicit checking of the request params on the import command; is there a reason to keep it here and not there?